dluhc-component-library/plan-making/assets/policy.json

```diff
@@ -1,7 +1,7 @@
 {
-  "reference": "1234",
-  "title": "test policy title",
-  "description": "this is a test policy",
+  "reference": "NE1234",
```

Comment:
is this meant to be a number?

Reply:
Its a string so can contain characters too.

Reply:
The type in `PolicyData` will need changing then

Reply:
I think `PolicyData` can be deleted altogether as its a duplication of the `FormState` in the same file. Probably just needs a rename.